docs/targets/googlecloudpubsub.md

```diff
@@ -0,0 +1,33 @@
+# Event Target for Google Cloud Pub Sub
```

Comment:
Here and below: the product name is "Google Cloud **Pub/Sub**", so let's stick to it.

```suggestion
# Event Target for Google Cloud Pub/Sub
```